tests/workflows/test_uber_lyft.py

```diff
@@ -0,0 +1,67 @@
+import uuid
+
+import coiled
+import dask.dataframe as dd
+import pytest
+from dask.distributed import Client
+
+
+@pytest.fixture(scope="module")
+def uber_lyft_cluster(
+    dask_env_variables,
+    cluster_kwargs,
+    github_cluster_tags,
+):
+    with coiled.Cluster(
+        f"uber-lyft-{uuid.uuid4().hex[:8]}",
+        environ=dask_env_variables,
+        tags=github_cluster_tags,
+        **cluster_kwargs["uber_lyft_cluster"],
+    ) as cluster:
+        yield cluster
+
+
+@pytest.fixture
+def uber_lyft_client(
+    uber_lyft_cluster,
+    cluster_kwargs,
+    upload_cluster_dump,
+    benchmark_all,
+):
+    n_workers = cluster_kwargs["uber_lyft_cluster"]["n_workers"]
+    with Client(uber_lyft_cluster) as client:
+        uber_lyft_cluster.scale(n_workers)
+        client.wait_for_workers(n_workers)
+        client.restart()
+        with upload_cluster_dump(client), benchmark_all(client):
+            yield client
+
+
+@pytest.fixture
+def ddf(uber_lyft_client):
+    """NYC taxi Uber/Lyft dataset"""
+    paths = [
+        f"s3://coiled-datasets/mrocklin/nyc-taxi-fhv/{year}-*.parquet"
```

Comment:
Couple of questions here
- I'm confused about the `year` arg. each of the `.parquet` in `"s3://coiled-datasets/mrocklin/nyc-taxi-fhv/` is a day-partitioned file

Is there a reason why we are creating a list of paths? you can read the whole dataset as  
```python
dd.read_parquet("s3://coiled-datasets/mrocklin/nyc-taxi-fhv/, engine="pyarrow")
```

- Is there a specific reason on why using this dataset? 
 
The partition sizes here are not of great size, they are quite small because it's partitioned  by day, and are kind of imbalanced a bit. 

`s3://coiled-datasets/uber-lyft-tlc/` this one is better partitioned (partition size ~100MB)

Reply:
Also, the dask-dataframe tutorial is based on this dataset. We are having a session on Wednesday in case you want to listen in and take a look and what this dataset looks like originally. The materials for reference are here https://github.com/coiled/dask-tutorial/blob/main/2-Get_better-at-dask-dataframes.ipynb, happy to chat about this dataset and workflow if needed.

Reply:
Thanks for the tip @ncclementi, I can switch to `s3://coiled-datasets/uber-lyft-tlc/`.

I think smaller partition sizes correlate with smaller worker sizes in @mrocklin 's example. If I use `s3://coiled-datasets/uber-lyft-tlc/`, I will probably need `m6i.xlarge` like you said.